exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h

```diff
@@ -15,6 +15,11 @@ struct OtlpExporterOptions
 {
   // The endpoint to export to. By default the OpenTelemetry Collector's default endpoint.
   std::string endpoint = "localhost:4317";
+  // By default when false, uses grpc::InsecureChannelCredentials(); If true,
+  // uses ssl_credentials_cacert_path
+  bool use_ssl_credentials = false;
```

Comment:
Is it possible to add a test case for `use_ssl_credentials = true`?

Reply:
added a test for config.